cli/src/pcluster/resources/imagebuilder/parallelcluster_validate.yaml

```diff
@@ -199,31 +199,17 @@ phases:
                 echo cuda-${cuda_ver}/samples/bin
               fi
 
-      - name: ArmPLVersion
-        action: ExecuteBash
-        inputs:
-          commands:
-            - |
-              set -v
-              PATTERN=$(jq '.default.cluster.armpl.major_minor_version' {{ CookbookDefaultFile }})
-              MAJOR_MINOR_VERSION=$(echo ${PATTERN} | tr -d '\n' | cut -d = -f 2 | xargs)
-              VERSION+="${MAJOR_MINOR_VERSION}."
-              PATTERN=$(jq '.default.cluster.armpl.patch_version' {{ CookbookDefaultFile }})
-              PATCH_VERSION=$(echo ${PATTERN} | tr -d '\n' | cut -d = -f 2 | xargs)
-              VERSION+="${PATCH_VERSION}"
-              echo ${VERSION}
-
-      - name: ArmPLGCCVersion
+      ### utils ###
+      - name: PatchInSpecProfiles
         action: ExecuteBash
         inputs:
           commands:
             - |
               set -v
-              PATTERN=$(jq '.default.cluster.armpl.gcc.major_minor_version' {{ CookbookDefaultFile }})
-              VERSION=$(echo ${PATTERN} | tr -d '\n' | cut -d = -f 2 | xargs)
-              echo ${VERSION}
+              sudo sed -i "s#path: cookbooks#path: /etc/chef/cookbooks#g" /etc/chef/cookbooks/aws-parallelcluster/test/recipes/inspec.yml
```

Comment:
I)'d add a comment here explaining WHY we're doing it (the same we wrote in the PR description is good).

Reply:
Will do it in one of the next PRs.